### PR TITLE
i18n(fr): translate turso.mdx

### PR DESCRIPTION
**Pull request by MartinFerret**
French translation of the turso.mdx

### CODE REVIEW

**Comment by MartinFerret** [comment]
@florian-lefebvre , est-ce normal que j'ai un failed au niveau du deploy Bervel - docs ?

**Comment by florian-lefebvre** [comment]
I'm not really active on this repo so I don't why that happens. Don't worry, if you need to do anything someone will tell while reviewing

**Comment by shuuji3** [comment]
Hi, I'm not a French translator but I think the cause of the error is because the frontmatter keys such as `title:` and `i18nReady` are translated.

This error message (https://vercel.com/astrodotbuild/docs/5NwrZMZhvB6f4CJq1cR1Bvx7Gs2D#L26) tells us that it cannot find an expected key `title:` in the header.

These headers have special meaning so when translating documentation, we should keep them in English though you can translate the right-hand text after `:`.

File: src/content/docs/fr/guides/backend/turso.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+titre: Turso & Astro
```
Comment (shuuji3) [inline src/content/docs/fr/guides/backend/turso.mdx:2]:
```suggestion
title: Turso & Astro
```

**Review by shuuji3 — CHANGES_REQUESTED** [review]
Could you try this change?

**Comment by MartinFerret** [comment]
> I also put a comment on #6023 🙂

Thanks a lot for commenting and helping ! 😀

**Review by VoxelMC — APPROVED** [review]
Ça a l'air génial ! Thank you Martin!

`As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!`

File: src/content/docs/fr/guides/backend/turso.mdx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Turso](https://turso.tech) est une base de données distribuée construite sur libSQL, un fork de SQLite. Elle est optimisée pour une faible latence de requête, la rendant adaptée aux applications globales.
+
+## Initialisation de Turso dans Astro
+
+### Prérequis
+
+- Le [Turso CLI](https://docs.turso.tech/reference/turso-cli) installé et connecté
```
Comment (McFlyPartages) [inline src/content/docs/fr/guides/backend/turso.mdx:17]:
```suggestion
- [Turso CLI](https://docs.turso.tech/reference/turso-cli) installé et connecté
```

File: src/content/docs/fr/guides/backend/turso.mdx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+Ajoutez le résultat des deux commandes ci-dessus dans votre fichier `.env` à la racine de votre projet. Si ce fichier n'existe pas, créez-le.
+
+```ini title=".env"
+TURSO_DATABASE_URL=libsql://...
+TURSO_AUTH_TOKEN=
+```
+
+:::attention
```
Comment (McFlyPartages) [inline src/content/docs/fr/guides/backend/turso.mdx:43]:
```suggestion
:::caution
```

It's not necessary to translate this

**Review by McFlyPartages — COMMENTED** [review]
Hey,

Thanks for good work, here are a few suggestions.

**Review by McFlyPartages — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work y'all! 💜

**Pull request merged**